site/content/entry/citra-progress-report-2017-p1.md

```diff
@@ -0,0 +1,33 @@
++++
+date = "2017-04-04T20:00:00-04:00"
+title = "Citra Progress Report - 2017 P1"
+tags = [ "progress-report" ]
+author = "anodium"
+forum = 00
++++
+
+It's a bit hard to believe, but Citra is already [three years old](https://github.com/citra-emu/citra/commit/7a7917df27cfe5c342bba6c15b57d389fce585bd), and in these three years we've gone from an emulator that could barely run homebrew, to one that can run many commercially available games at playable speeds! It's all thanks to the hard work of volunteers working to improve the emulator at a pace we could've only imagined back in the spring of 2014, and it seems 2017 shows no signs of slowing down. We've had a lot of improvements so far with accuracy and even more 3DS features, so let's take a look.
```

Comment:
Gods this sounds *so* much better, definitely adding that, since I really just put that sentence in there to essentially shove people into the meat of the article.

@JayFoxRox Also suggested to me in DMs that I invert the paragraph structure, such that "It's...speeds!" and "It's...down." are swapped. (Correct me if I interpreted your feedback wrong.)

> [6:40 PM] JayFoxRox: I think it would be better to start out with saying that 2017 is not showing signs of slowing down, many contributors [maybe also mention newbs if we had any this year?] and THEN congratulating on the 3 years
> [6:40 PM] JayFoxRox: but I'll probably try to come up with something tomorrow when I do the full review
> [6:41 PM] JayFoxRox: because right now the headline doesn't match the first sentence so my first impression was like: "wait.. is a sentence missing here? is this a full 3 year recap or just for 2017? wait... 2014? wat?"